### PR TITLE
Feature/engarde 4.2

### PR DESCRIPTION
**Pull request by TudoruData**

### CODE REVIEW

**Comment by TudoruData** [comment]
Made a mistake when rebasing, causing duplicate commit record, but don't see any problems in the actual code

**Review by theSkyseS — APPROVED** [review]

File: Engarde_Synthesis/Engarde_Synthesis.csproj [hunk 1]
```diff
@@ -11,8 +11,4 @@
     <PackageReference Include="Mutagen.Bethesda.Synthesis" Version="0.18.2" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Folder Include="FormKeys" />
-  </ItemGroup>
-
```
Comment (theSkyseS) [inline Engarde_Synthesis/Engarde_Synthesis.csproj:17]: Probably unintended change?

Comment (TudoruData) [reply Engarde_Synthesis/Engarde_Synthesis.csproj:17]: VS studio made this change, I'm not exactly sure what this change does. Do you know?

Comment (theSkyseS) [reply Engarde_Synthesis/Engarde_Synthesis.csproj:17]: I actually don't know, but if it works without these lines, then let them be deleted

**Pull request merged**